### PR TITLE
Student Responses Data page

### PR DESCRIPTION
**Pull request by Enoch-Hsiao**
**Data/Student responses page:**

- Table: Allows for filtering, export as csv file
- StudentResponseDialog: Allows user to view action decisions, reflection responses, (and eventually radar plot)

DataFlow

### CODE REVIEW

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/.eslintrc.json [hunk 1]
```diff
@@ -29,5 +29,6 @@
      * "error" or 2 - turn the rule on as an error (exit code will be 1)
      * There could be other rules not added below that should be added
      */
+     "react/display-name": "off"
```
Comment (eneadodi) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/.eslintrc.json:32]: what does this do?

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/EditorComponents/ActionPageComponents/Action.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ const useStyles = makeStyles((theme) => ({
     },
 }));
 
-export default function FinalAction(props) {
+export default function Action(props) {
```
Comment (eneadodi) [inline Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/EditorComponents/ActionPageComponents/Action.js:33]: Was this change a remnant of old namings? or is this something we should be concerned about?

**Review by eneadodi — APPROVED** [review]
Possibly the most impressive PR yet. Very well done!

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/.eslintrc.json (hunk 1 above)
Comment (Enoch-Hsiao) [reply Ghost_in_the_Shell/client_folder_react_app/ethisim/.eslintrc.json:32]: I had eslint errors for this, I don't think it's important

File: Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/EditorComponents/ActionPageComponents/Action.js (hunk 2 above)
Comment (Enoch-Hsiao) [reply Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/EditorComponents/ActionPageComponents/Action.js:33]: yea, should be called Action component, shouldn't affect anything

**Pull request merged**